source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -112,10 +112,9 @@ content: |
 
            final BsonString masterKeyRegion = new BsonString("<Master Key AWS Region>"); // e.g. "us-east-2"
            final BsonString masterKeyArn = new BsonString("<Master Key ARN>"); // e.g. "arn:aws:kms:us-east-2:111122223333:alias/test-key"
-           DataKeyOptions dataKeyOptions = new DataKeyOptions().masterKey(new BsonDocument() {{
-               put("region", masterKeyRegion);
-               put("key", masterKeyArn);
-           }})
+           DataKeyOptions dataKeyOptions = new DataKeyOptions().masterKey(new BsonDocument();
```

Comment:
Make sure you try to compile all the example.  This one doesn't compile.  I think you meant:

        DataKeyOptions dataKeyOptions = new DataKeyOptions().masterKey(
                new BsonDocument()
                        .append("region", masterKeyRegion)
                        .append("key", masterKeyArn));

Reply:
Oops, thanks for catching that. 😬 Will fix.